src/components/Home/Home.tsx

```diff
@@ -0,0 +1,88 @@
+import { useQuery } from "@tanstack/react-query";
+import { ProductList } from "@/components/ProductList";
+import { getProducts } from "@/api/getProducts";
+import { Product } from "@/components/Product";
+import { SearchBar } from "../SearchBar";
+import { useState } from "react";
+import { SortBy } from "../SortBy";
+import { ProductSchema } from "@/schemas/ProductSchema";
+import { SortByValues } from "@/enums/sortByValues";
+import { sortAlphabetically } from "@/utils/sortAlphabetically";
+
+export const Home = () => {
+  const { data: products } = useQuery({
+    queryKey: ["getProducts"],
+    queryFn: getProducts,
+  });
+
+  const [filter, setFilter] = useState<string | undefined>();
+  const [sortBy, setSortBy] = useState<string | undefined>();
+  const [favoriteProducts, setFavoriteProducts] = useState<string[]>([]);
+
+  const handleFilters = (): ProductSchema[] => {
+    if (products) {
+      let sortedFilteredProducts: ProductSchema[] = products;
+      if (sortBy) {
```

Comment:
I think this `if` can be removed, and the result will be the same since all other if's will fail